recipes/pdir2/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pdir2" %}
+{% set version = "0.1.1" %}
+{% set hash_type = "sha256" %}
+{% set hash = "947b374d679bc777385325ad3a24693c9a3c13e4c6854fda09b5061ae5021332" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
Done, my bad I didn't review that section more carefully after using `skeleton pypi`.